include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -610,4 +610,17 @@ def HWStructCastOp : FIRRTLOp<"hwStructCast", [NoSideEffect]> {
     "$input attr-dict `:` functional-type($input, $result)";
 
   let verifier = "return ::verifyHWStructCastOp(*this);";
-}
\ No newline at end of file
+}
+
+def BitCastOp: FIRRTLOp<"bitcast", [NoSideEffect]> {
+  let summary = [{
+    Reinterpret one value to another value of the same size and
+    potentially different type. This op is lowered to hw::BitCastOp.
+  }];
+
+  let arguments = (ins AnyType:$input);
+  let results = (outs AnyType:$result);
```

Comment:
This should use FIRRTLType instead of AnyType for both things.

Reply:
Thanks, this was a typo.